report_py3o/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Py3o Report Engine",
+    "summary": "Reporting engine based on Libreoffice (ODT -> ODT, "
+    "ODT -> PDF, ODT -> DOC, ODT -> DOCX, ODS -> ODS, etc.)",
+    "version": "17.0.1.0.0",
+    "category": "Reporting",
+    "license": "AGPL-3",
+    "author": "XCG Consulting, ACSONE SA/NV, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/reporting-engine",
+    "depends": ["web"],
+    "external_dependencies": {
+        "python": ["py3o.template", "py3o.formats"],
+        "deb": ["libreoffice"],
```

Comment:
More info: https://github.com/odoo/odoo/blob/d759264c6e9b7a8cc8b6cffdc5509df654cd434c/odoo/modules/module.py#L508

```suggestion
        "bin": ["libreoffice"],
```

I think this should be changed on all versions... and other modules...

Reply:
[odoo docs](https://www.odoo.com/documentation/17.0/developer/reference/backend/module.html?highlight=external_dependencies) are pretty clear on this. Nice catch.

Reply:
Umm... looks like OCA "unofficial" way (i don't know where is documented) to handle dependencies in CI env: https://github.com/OCA/oca-ci/blob/c10c90835d3baf6576faaae03f593099b9995009/bin/oca_install_addons#L16

Perhaps a solution could be to use 'bin' to have the Odoo check and 'apt' to indicate how to install the package on X distro?

ping @sbidoul

Reply:
You can add `bin` if you want, but please keep `deb` so libreoffice is installed in ci and runboat.

That said, I'm not sure libreoffice is a hard dependency. IIRC it's optional when using `report_py3o_fusion_server`, so in that case we should not add `bin`.

Reply:
@bug2point0 I think there should be both “bin” and “deb”. 
As I see it, actually it all depends on which module you give more importance to 'report_py3o_fusion_server' vs 'report_py3o'... Everything else is a mess of dependencies (if you choose X with Y but you have Z then disable W... :/). At least from what I could see.

You decide, if you choose only “deb” it seems right to me... and if you choose “bin” and “deb” the same. Little more I can say.